transfer/client.go

```diff
@@ -16,8 +16,14 @@ const (
 	Failed   stripe.TransferStatus = "failed"
 	Canceled stripe.TransferStatus = "canceled"
 
-	Card stripe.TransferType = "card"
-	Bank stripe.TransferType = "bank_account"
+	Card          stripe.TransferType = "card"
+	Bank          stripe.TransferType = "bank_account"
+	StripeAccount stripe.TransferType = "stripe_account"
+
+	SourceAlipay  stripe.TransferSourceType = "alipay_account"
+	SourceBank    stripe.TransferSourceType = "bank_account"
+	SourceBitcoin stripe.TransferSourceType = "bitcoin_receiver"
+	SourceCard    stripe.TransferSourceType = "card"
```

Comment:
I'm still overall a little uncertain about the ideal naming/location of constants like these ones. Obviously the transfer type constants are already in here, but we can see similar constants from other resources, [PaymentSourceType](payment_source.go) for example, located right at the top level of the package.

What do you think about giving these a full `TransferSource` type prefix for now just as a hedge against future re-organization? (The current `Source` fits the naming in `Transfer`, but could be read as a little generic.)